nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -262,8 +269,7 @@ private void printExecutionResults(Set<String> allFailures, CleanSurefireExecuti
 
     private void writeCurrentRunInfo(CleanSurefireExecution execution) {
         try {
-            // TODO(gyori): This is quite ugly, you grabbing here the first from a list to establish a run id.
-            Files.write(this.executions.get(0).getConfiguration().getRunFilePath(),
+            Files.write(this.runFilePath,
```

Comment:
Since this function is called from multiple places, is it possible for `this.runfilePath` to be null, thereby causing NPE?

Reply:
According to the professor,and TA, The NonDex PR's needs to be reviewed only by professor and TA. Please do not review Nondex PR. 

For the comment, the `writeCurrentRunInfo` file is called in two places 

https://github.com/kavvya97/NonDex/blob/8843b436802af4a0850f63d6ae0f231164dafe78/nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java#L97-L101

The runfile path is set within the same function before any of these calls. so It cannot be null.